### DISCUSSION TITLE
PLEASE HELP ME I HAVE A HOMEWORK!! How can i develop a 3d liver segmentation model?

### DISCUSSION THREAD

**Discussion by TunaElma** [Project-MONAI/MONAILabel#1642] [General]
Hello team!
I have a homework for monday. I should develop a 3d liver segmentation model but i couldnt. I installed monailabel, 3d slicer. I can use pretrained models but i couldnt develop my own model. Can someone help me in detail?

**Comment by diazandr3s** [comment]
Hi @kahraman1127,

I'd suggest you follow these steps for training a MONAI Label model from scratch: https://youtu.be/3HTh2dqZqew?si=AyEYz7-YSV-zEcH2

and use the liver dataset from the MSD: http://medicaldecathlon.com/

Here you can find more information about MONAI Label: https://youtu.be/-HAryYAO5J4?si=ZuJvVyEbipw_3ngR&t=150

I hope this helps,
